### PR TITLE
Ignore null ExpiresUtc property in RequestRefresh

### PR DESCRIPTION
**Pull request by troydai**
Issue: https://github.com/aspnet/Security/issues/949

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 1]
```diff
@@ -57,11 +57,14 @@ private void CheckForRefresh(AuthenticationTicket ticket)
 
         private void RequestRefresh(AuthenticationTicket ticket)
         {
-            _shouldRefresh = true;
-            var currentUtc = Options.SystemClock.UtcNow;
-            _refreshIssuedUtc = currentUtc;
-            var timeSpan = ticket.Properties.ExpiresUtc.Value.Subtract(ticket.Properties.IssuedUtc.Value);
-            _refreshExpiresUtc = currentUtc.Add(timeSpan);
+            if (ticket.Properties.ExpiresUtc != null)
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: .HasValue?

File: src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 2]
```diff
@@ -57,11 +57,14 @@ private void CheckForRefresh(AuthenticationTicket ticket)
...
+            {
+                _shouldRefresh = true;
+                var currentUtc = Options.SystemClock.UtcNow;
+                _refreshIssuedUtc = currentUtc;
+                var timeSpan = ticket.Properties.ExpiresUtc.Value.Subtract(ticket.Properties.IssuedUtc.Value);
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: What about IssuedUtc?

Comment (troydai) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: Covered in the last commit.

File: src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 3]
```diff
@@ -57,11 +57,14 @@ private void CheckForRefresh(AuthenticationTicket ticket)
 
         private void RequestRefresh(AuthenticationTicket ticket)
         {
-            _shouldRefresh = true;
-            var currentUtc = Options.SystemClock.UtcNow;
-            _refreshIssuedUtc = currentUtc;
-            var timeSpan = ticket.Properties.ExpiresUtc.Value.Subtract(ticket.Properties.IssuedUtc.Value);
-            _refreshExpiresUtc = currentUtc.Add(timeSpan);
+            if (ticket.Properties.ExpiresUtc.HasValue && ticket.Properties.IssuedUtc.HasValue)
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: These properties parse, store them in local variables.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**